_guide/_pages/phone-screens.md

```diff
@@ -59,70 +61,128 @@ Begin by introducing yourself by saying this or something similar to it:
 
 ### Discuss the candidate’s career history and path
 
-Give the candidate space to tell their story, so we can hear how they view themselves and where they are heading in their career.
+> Give the candidate space to tell their story, so we can hear how they view themselves and where they are heading in their career.
 
-- "Tell me your professional story."
-  - "Tell me about a few of your recent projects."
-- "What got you interested in 18F?"
+- "I’ve had the opportunity to take a look at your resume, but I’d love to hear in your own words a brief version of your professional story."
+- "Tell me about a few of your recent projects."
 - "What are you looking for in your next ideal role, and what things are most important for you?"
+- "What got you interested in TTS?"
```

Comment:
I slacked you a product-y question which might relate to the usage of the name 18F or TTS in this question, just flagging the location here for quick reference if needed.